src/tools/rust-analyzer/crates/parser/src/lexed_str.rs

```diff
@@ -177,6 +177,16 @@ impl<'a> Converter<'a> {
                     COMMENT
                 }
 
+                #[cfg(feature = "in-rust-tree")]
```

Comment:
This cfg shouldn't be necessary. We will publish an updated version of the lexer crate when we do the next sync

Reply:
Thanks for reviewing the r-a changes :)

Before the next sync, this match arm will fail to compile (at least from the Rust tree using `x c` IIRC). I had thought that this is how we can annotate such things such as extra variants before a sync happens (and when the sync happens the `cfg` attribute gets removed). Would you like me to check again whether removing it still compiles, or is there an alternative here?

Reply:
Thanks, I've removed the cfg.